presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -498,9 +498,9 @@ protected Type toPrestoType(int jdbcType, int columnSize)
             case Types.DATE:
                 return DATE;
             case Types.TIME:
-                return TIME;
+                return "timetz".equalsIgnoreCase(typeName) ? TIME_WITH_TIME_ZONE : TIME;
```

Comment:
Are "timetz" and "timestamptz" applicable to any database (i.e., defined by the JDBC spec?) or are they specific to postgresql? If the latter, this should go in `PostgreSqlClient`.

Reply:
In netezza, timetz that is alias time with time zone is OK.
https://www.ibm.com/support/knowledgecenter/en/SSULQD_7.2.1/com.ibm.nz.sproc.doc/c_sproc_data_types_aliases.html